.github/workflows/bencher.yml

```
@@ -0,0 +1,112 @@
+      - name: Run benches → BMF
+        run: pnpm -s bench:bmf
+
+      - name: Install Bencher CLI
+        if: env.BENCHER_API_TOKEN != ''
+        uses: bencherdev/bencher@main
+
+      # Fork PRs have no access to BENCHER_API_TOKEN, so this step is skipped
+      # for them. The bench artifacts can still be uploaded for inspection.
+      - name: Upload to Bencher (PR)
+        if: env.BENCHER_API_TOKEN != ''
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)